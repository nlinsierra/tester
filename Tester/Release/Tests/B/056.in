5D 5H 4S TC JC 4H QH 6S JH JS 9S AD QD 2H 6H 8C QC 2D 9D 9H JD 7S 6D 8S TS KH 5C AH QS AC 3H KS 4D 7C 3S KC TH 3C
