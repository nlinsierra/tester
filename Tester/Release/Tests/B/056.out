14
2C 4C 6C 9C 3D 7D 8D TD KD 7H 8H 2S 5S AS
